### PR TITLE
AVC denials check for tests

### PR DESCRIPTION
**Pull request by happz**
First real post-test check, complementing whatever the test itself did
by checking the system for AVC denials raised during the test runtime.

Follows the design established by Restraint's AVC denial check we all
know and love, and performs the same tasks.

### CODE REVIEW

File: tmt/checks/avc.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                output.stdout or '',
+                ''
+                ]
+
+        else:
+            report += [
+                '# sestatus',
+                "\n".join(render_run_exception_streams(exc.stdout, exc.stderr, verbose=1)),
+                ''
+                ]
```
[the remark this answers is not in the document]
Comment (happz) [reply tmt/checks/avc.py:105]: Yeah, that's what I was thinking too, it looks awfully similar, right? But it turned out the generalization did not shorten the code enough, and the special handling of denials required the third special case to exist anyway, so I gave up :/ But I can look at it once again, I agree there are similarities.

File: tmt/checks/avc.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            got_denials = True
+
+            report += [
+                '# ausearch',
+                "\n".join(render_run_exception_streams(output.stdout, output.stderr, verbose=1)),
+                ''
+                ]
+
+        elif exc.returncode == 1 and exc.stderr and '<no matches>' in exc.stderr.strip():
+            got_ausearch = True
```
[the remark this answers is not in the document]
Comment (happz) [reply tmt/checks/avc.py]: Done in 89337611

File: tmt/checks/avc.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (happz) [reply tmt/checks/avc.py]: Done in 89337611

**Review by KwisatzHaderach — APPROVED** [review]

**Review by psss — APPROVED** [review]
Very nice! Thanks for implementing this. I just suggest to rename the class in 9387cf52 for consistent naming, if you don't mind.

**Comment by happz** [comment]
Ack, go ahead.

**Pull request merged**